cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -563,9 +599,13 @@ public void setupCCStopResponse() {
     }
 
     private Parameter buildBrokerIdParameter(CruiseControlEndpoints endpoint) {
-        return CruiseControlEndpoints.ADD_BROKER.equals(endpoint) || CruiseControlEndpoints.REMOVE_BROKER.equals(endpoint) ?
-                Parameter.param(CruiseControlParameters.BROKER_ID.toString(), "[0-9]+(,[0-9]+)*") :
-                null;
+        if (CruiseControlEndpoints.ADD_BROKER.equals(endpoint) || CruiseControlEndpoints.REMOVE_BROKER.equals(endpoint)) {
+            return Parameter.param(CruiseControlParameters.BROKER_ID.toString(), "[0-9]+(,[0-9]+)*");
+        } else if (CruiseControlEndpoints.REMOVE_DISKS.equals(endpoint)) {
```

Comment:
you are adding this here but then the method name is wrong, or?

Reply:
Yep, I should rename this to make it more generic which supports all 3(remove-broker, add-broker, remove-disks) endpoint